src/components/GestureButtons.tsx

```diff
@@ -92,6 +92,11 @@ export interface BaseButtonProps extends RawButtonProps {
    * Defaults to 600.
    */
   delayLongPress?: number;
+
+  /**
+   * Used to pass ref from parent into `BaseButton`
+   */
+  innerRef?: React.ForwardedRef<unknown>;
```

Comment:
Why not `React.ForwardedRef<React.ComponentType<any>>` since it's being cast anyway?

Or, if that's not possible, simply `React.ForwardedRef<any>`, since there is `any` in the casted type anyway?

Reply:
Makes sense, done in [fec6943](https://github.com/software-mansion/react-native-gesture-handler/pull/2903/commits/fec69434e1643e874c85b2122948fe480991e0c9)